### ISSUE TITLE
Including product tax category in the notification emails

### ISSUE THREAD

**Issue by NickWeir63** [openfoodfoundation/openfoodnetwork#920]
how big a job would it be to include the tax category in the notification emails?  As we are producing Quickbooks invoices for Stroudco shoppers as soon as the order confirmation email goes out, it would help Oliver a lot to be able to see straight away which items are VAT-able and which aren't without getting a report up.

If it can't be done easily, we'll either try and tell from memory or we might mark all VAT items with "(v)" in the product description.

**Comment by lin-d-hop** [comment]
Just a note that it will be important to have the three tax codes specified:
Standard
Exempt / Zero rated
Reduced Rate

**Comment by NickWeir63** [comment]
@daniellemoorhead please can you give us an idea of how much dev time this would take.  thanks

**Comment by lin-d-hop** [comment]
@NickWeir63 This should be straight forward if all products have their correct tax category set. Could you confirm that Stroudco are now using the correct tax categories and not relying on Enterprise Fees to specify tax?

**Comment by lin-d-hop** [comment]
@daniellemoorhead UK can handle this one :-)

**Comment by NickWeir63** [comment]
Good point Lynne - I will ask @oliver to confirm where he is up to with tax categories so that we can pick up VAT totals accurately.  He is away until tomorrow.
PS when I type @oliver it doesn't bring up his github account name like it used to........

**Comment by OliverUK** [comment]
@NickWeir63  Not sure why I don't come up. It should be OliverUK but usually it finds people for you.
I've been using the tax categories for each product so if this can be used for the emails, great! However last time I heard we still needed to use separate enterprise fees because we are not getting the right end price otherwise. @lin-d-hop

**Comment by stveep** [comment]
Are you talking about the same email as #927 here?

**Comment by NickWeir63** [comment]
yes this is the same email

**Comment by OliverUK** [comment]
@stveep  @NickWeir63 
Actually, no, it's not.
Here we are talking about the order confirmation sent to customers and copied to the hub. As the hub manager I would like to be able to see on that email which items are subject to VAT.

The email at #927 is the purchase order to suppliers, which needs a total amount at the end and ideally a more readable layout and customisable content.

**Comment by NickWeir63** [comment]
Sorry - my mistake.  Yes these are 2 separate emails.  So @stveep shall we assign this issue to someone else?
I think @OliverUK would agree that this is a less urgent issue than #927

**Comment by OliverUK** [comment]
Hi Everyone, 
@NickWeir63 @lin-d-hop  @stveep 

For the time being, Stroudco is **deregistering from VAT**. 

I apologise for any wasted time on your side - bloody users, right? 

For this reason we don't need the above feature anymore, at least until such time when we may register again in the future - or unless we are told for whatever obscure reason that the tax man won't let us deregister.

**Comment by stveep** [comment]
The producer notification emails now have tax (amount but not category) included. It shouldn't be too hard to add them to this email as well. As far as I can see these emails come straight from native Spree, so if you don't need tax in these now I suggest waiting for the Spree upgrade, as this should improve the emails (a bit - at least they will no longer be plain text) and provide a better starting point for anything we need to add.

**Comment by stveep** [comment]
Just discussing this in our Skype - @RohanM do you know if the Spree upgrade will affect this (see comment above)?

**Comment by daniellemoorhead** [comment]
@RohanM are you able to answer @stveep's question from June 30?

**Comment by RohanM** [comment]
Hi @stveep,

I don't have a definite yes or no on that one (without looking into it further). I doubt that the upgrade will automatically split the tax into categories on that email, but the improvements to the data model might make it easier to do the calculation and get the correct data. Up to you whether to build this now or later depending on the urgency.

**Comment by kirstenalarsen** [comment]
@enricostano @oeoeaio @RohanM  is this issue related to / should be considered in Spree Upgrade? or is it obsolete and can be closed? Opinions from @stveep @NickWeir63 and @OliverUK also welcome!

**Comment by enricostano** [comment]
cc/ @sauloperez

**Comment by sauloperez** [comment]
What I see is that as of https://github.com/spree/spree/commit/cf3be8e44 line items include their tax category so it's easy to add in the order confirmation email. So it's better to wait for the upgrade.

**Comment by oeoeaio** [comment]
@sauloperez do we get that commit in step 6 or step 7?

**Comment by sauloperez** [comment]
will have to wait until step 7 to get it.

**Comment by oeoeaio** [comment]
👍

**Comment by luisramos0** [comment]
Removing the reference to #2109 and moving it to epic #2584 (all the issues blocked by the spree upgrade).

**Comment by RachL** [comment]
@lin-d-hop is that still something needed?
